### PR TITLE
rt refine: remove BlockedOnReply from simple'

### PR DESCRIPTION
**Pull request by corlewis**

### CODE REVIEW

**Review by lsf37 — APPROVED** [review]
That is amazingly little change!

**Comment by corlewis** [comment]
> 
> 
> That is amazingly little change!

Yeah, and they're actually weren't any changes needed at all! What's here is just some cleanup that became possible after removing it.

File: proof/refine/ARM/IpcCancel_R.thy [hunk 1]
```diff
@@ -105,10 +105,10 @@ lemma cancelSignal_st_tcb_at':
                     hoare_drop_imp[where R="\<lambda>rv _. \<exists>a b. delete t (f rv) = a # b" for f])
   done
 
-lemma cancelIPC_simple'_not_awaiting_reply:
+lemma cancelIPC_simple':
   "\<lbrace>\<lambda>s. sym_refs (state_refs_of' s)\<rbrace>
    cancelIPC t
-   \<lbrace>\<lambda>_. st_tcb_at' ((=) Running or (=) Inactive or (=) Restart or (=) IdleThreadState) t\<rbrace>"
```
Comment (mktnk3) [inline proof/refine/ARM/IpcCancel_R.thy:111]: Ah, right, this is the result of the previous discussion. And now we are aligning `simple’` with it, which is good!

**Review by mktnk3 — APPROVED** [review]

**Review by michaelmcinerney — APPROVED** [review]
!!!!

**Review by MitchellBuckley — APPROVED** [review]
Wow! Very small change indeed!

**Pull request merged**